domain/src/main/java/org/oppia/domain/question/QuestionProgressController.kt

```diff
@@ -0,0 +1,18 @@
+package org.oppia.domain.question
+
+import org.oppia.app.model.Question
+import javax.inject.Inject
+import javax.inject.Singleton
+
+
+/** Controller for retrieving an exploration. */
+@Singleton
+class QuestionProgressController @Inject constructor(
```

Comment:
I was intending to call this ``QuestionAssessmentProgressController`` per #111, since a single 'question' doesn't really have progress. Please update.

Reply:
Done